src/Serializer/AbstractSerializer.php

```diff
@@ -241,7 +241,14 @@ protected function serializeValue($value)
         }
 
         if (\is_object($value)) {
-            return 'Object ' . \get_class($value);
+            $objectId = null;
+            if (isset($value->id)) {
+                $objectId = $value->id;
+            } elseif (method_exists($value, 'getId')) {
```

Comment:
If you're trying to call an arbitrary method on unknown object I think you'd have to use `is_callable()` to ensure you have access to the method?